File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/iget-person-vetting_item.ts




export interface IGetPersonVetting_Item {
  
	PersonsVettingID: number;
	PersonID: number;
	VettingPersonStatusID: number;
	VettingStatus: string;
	BatchID: string;
	TrackingNumber: string;
	VettingValidStartDate?: Date;
	VettingValidEndDate?: Date;

}

File: GTTS/INL.PersonService.Models/IGetPersonVetting_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.PersonService.Models
{
    public interface IGetPersonVetting_Item
    {
        long PersonsVettingID { get; set; }
        long PersonID { get; set; }
        int VettingPersonStatusID { get; set; }
        string VettingStatus { get; set; }
        string BatchID { get; set; }
        string TrackingNumber { get; set; }
        DateTime? VettingValidStartDate { get; set; }
        DateTime? VettingValidEndDate { get; set; }
    }
}
